{"uuid":"dcd6dfb5-aba8-4d4c-82f8-8a8539cdc6f4","historyId":"6e48c4d2a9120abab5d7184da7559d83","fullName":"org.example.web.tests.LoginTest.test_VWOLogin_Negative","labels":[{"name":"package","value":"org.example.web.tests.LoginTest"},{"name":"testClass","value":"org.example.web.tests.LoginTest"},{"name":"testMethod","value":"test_VWOLogin_Negative"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"SeleniumFrameworkFinal"},{"name":"subSuite","value":"org.example.web.tests.LoginTest"},{"name":"host","value":"MSI"},{"name":"thread","value":"15908@MSI.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"critical"}],"links":[],"name":"test_VWOLogin_Negative","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Cannot invoke \"org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)\" because the return value of \"org.example.driver.driverManagerTL.getDriver()\" is null","trace":"java.lang.NullPointerException: Cannot invoke \"org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)\" because the return value of \"org.example.driver.driverManagerTL.getDriver()\" is null\r\n\tat org.example.utils.ScreenShotListener.onTestFailure(ScreenShotListener.java:24)\r\n\tat org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:108)\r\n\tat org.testng.internal.invokers.TestInvoker.runTestResultListener(TestInvoker.java:262)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:751)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","description":"Verify that invalid login credentials are not working","steps":[],"attachments":[],"parameters":[],"start":1695995009167,"stop":1695995009167}